Backend/app.js

```diff
@@ -1,19 +1,29 @@
 const express = require('express');
 const app = express();
 const models = require('./models');
-const router = express.Router();
-const rootRouter = require('./router/index');
 const bodyParser = require('body-parser');
+const router = express.Router();
+const session = require('express-session');
 
 models.sequelize.sync();
 
+app.use(session({
+  secret: 'asadlkj!a@#a!@a#dfgasdg',
```

Comment:
what is the secret here?

Reply:
passwords like this should be stored in a separate file (and read in at server loading time), which is excluded from git by .gitignore. :)